roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/util/dto/Track.java

```diff
@@ -0,0 +1,26 @@
+package at.fhv.itm3.s2.roundabout.util.dto;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+public class Track {
```

Comment:
Whats the difference between track and streetsection?

Reply:
track is the name in xml, streetsection is the class we use in our model